dbt/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "dbt"
+version = "0.0.1"
```

Comment:
You unfortunately _have_ to set a version key, even if it's not really applicable to your project. I just set all of ours to `0.0.1`.